### ISSUE TITLE
Create separate tracking settings for during trips vs not during trips

### ISSUE THREAD

**Issue by aaronpk** [aaronpk/Overland-iOS#139]
Instead of hard-coding the during-trip settings (#54), I think this might be better as a totally separate settings for the user.

I am thinking about adding a new tab to the bottom for trip settings, where you'd be able to have a separate set of all of the iOS and Overland settings that apply while a trip is active.

My use case is that I want super high resolution data during trips (1s, no discarding points based on distance, "Nav" accuracy, etc. But when I am not moving, I don't need a point written every 1 second, I would rather have that something more like 1 or 2 minutes. I don't want to disable tracking entirely, since I do want the data that I've been in one location for some period of time.

Looking through the settings, I think I would keep the main settings page mostly as is, but create a new one called "Trip Settings" with the following:

iOS Settings
* Desired Accuracy
* Activity Type
* Show Background Indicator (On/Off)

Overland Settings
* Logging Mode
* Locations per Batch
* Discard Points Closer Than
* Min Time Between Points
* Prevent Screen Lock

Internally, when a trip starts, it would remove significant change updates so that setting doesn't prevent updates from being delivered in the background.

Changes to the main settings screen

* Show Background Indicator (On/Off) instead of the 3-state toggle, since the "during trips" setting moves to the trip settings page
* Remove "Prevent Screen Lock" setting since it moves to the trip settings

So basically there are a subset of settings that apply to trips, any settings there would override the main settings when a trip is active.

I'm curious about other peoples' thoughts on this!
